ci/environment.yml

```diff
@@ -1,11 +1,14 @@
 channels:
   - conda-forge
+  - pytorch
```

Comment:
Oh, also do we need the `pytorch` channel here, or is `conda-forge` sufficient?

Reply:
I believe so, [pytorch quickstart](https://pytorch.org/get-started/locally/) has a nifty tool which explicitly states the channels, and skimming anaconda it seems packages like pytorch-cuda is only on the pytorch channel.

Reply:
nvm, I retract that, as we're only installing pytorch which is on conda-forge
https://github.com/coiled/benchmarks/pull/787/commits/7c5b94e2132d2452814cd1ca8c0ab5e5dc122906